### DISCUSSION TITLE
[QoL] Change the file path message in "romfs/arc" warning

### DISCUSSION THREAD

**Discussion by jnguyen143** [Raytwo/ARCropolis#466] [Ideas] [answered]
In version 3.10.0 of ARCropolis, a message now displays if the romfs/arc directory is detected. However, the actual message writes it as "rom:/arc", which can be misleading for users as it is not the true file path to check.

I would like to suggest replacing "rom:/arc" with "sd:/atmosphere/contents/01006A800016E000/romfs/arc/", as the latter represents the full path of where to check for the directory, and therefore makes it easy to find and remove to prevent the warning from reappearing.

**Comment by Raytwo** [comment, marked as answer]
Done!

**Comment by Tac24** [comment]
why was this changed to no longer be compatible?
